src/routers/filing.py

```diff
@@ -114,4 +114,10 @@ async def get_contact_info(request: Request, lei: str, period_name: str):
 @router.put("/institutions/{lei}/filings/{period_name}/contact-info", response_model=ContactInfoDTO)
 @requires("authenticated")
 async def put_contact_info(request: Request, lei: str, period_name: str, contact_info: ContactInfoDTO):
-    return await repo.update_contact_info(request.state.db_session, lei, period_name, contact_info)
+    result = await repo.get_filing(request.state.db_session, lei, period_name)
+    if result:
+        return await repo.update_contact_info(request.state.db_session, lei, period_name, contact_info)
+    return JSONResponse(
+        status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
+        content=f"The LEI ({lei}) and period ({period_name}) that was attempted to be updated does not exist.",
```

Comment:
Small nit, but it's more that a Filing hasn't been started for that LEI and period.  The period may very well exist, and the LEI may exist, but just no filing started in this service.  So something like "There is no Filing for LEI {lei} in period {period_name}" or something along those lines that let's them know they need to create a Filing for the lei + period.